packages/playback-core/src/text-tracks.ts

```diff
@@ -93,13 +93,16 @@ export function setupTextTracks(
     });
   });
 
-  const forceHiddenThumbnails = () => {
+  const forceHiddenTracks = () => {
     // Keeping this a forEach in case we want to expand the scope of this.
     Array.from(mediaEl.textTracks).forEach((track) => {
       if (['subtitles', 'caption'].includes(track.kind)) return;
-      if (track.label !== 'thumbnails') return;
+      if (track.label !== 'thumbnails' && track.kind !== 'chapters') return;
```

Comment:
suggestion(non-blocking): these are logically equivalent (see [De Morgan's Laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws)) but this feels clearer/more expressive of the semantics here imo (take it or leave it)
```suggestion
      if (!(track.label === 'thumbnails' || track.kind === 'chapters')) return;
```

Reply:
I considered this :) sure, let's do the more readable